templates/decisions/decisionView.html.twig

```diff
@@ -46,60 +46,67 @@
-				{% include "components/_timeline.html.twig" %}
-				<div class="d-flex justify-content-around">
-					<div>
-						<h3 class="text-primary fw-bold text-center">Personnes impactées</h3>
-						<div class="d-flex justify-content-center my-3">
-							{% if impactedUsers %}
-								{% for impactedUser in impactedUsers %}
-									<img class="arrow profil-icons" src="{% if impactedUser.user.poster %}{{ asset('uploads/avatar/')~ impactedUser.user.poster }}{% else %}{{ asset('build/images/icons/person.svg') }}{% endif %}" alt="User profile pictures">
-								{% endfor %}
-							{% else %}
-								Aucun salarié n'est impacté par cette décision pour l'instant.
-							{% endif %}
+				<div class="container rounded-4 border shadow bg-light">
+					<h2 class="fs-2 fw-bold py-3"><img class="me-1" src="{{ asset('build/images/icons/calendar.svg') }}" alt="Icone de la section dates à retenir">Dates à retenir</h2>
+					{% include "components/_timeline.html.twig" %}
+					<div class="d-flex justify-content-around">
+						<div>
+							<h3 class="text-primary fw-bold text-center">Personnes impactées</h3>
+							<div class="d-flex justify-content-center my-3">
+								{% if impactedUsers %}
+									{% for impactedUser in impactedUsers %}
+										<img class="arrow profil-icons" src="{% if impactedUser.user.poster %}{{ asset('uploads/avatar/')~ impactedUser.user.poster }}{% else %}{{ asset('build/images/icons/person.svg') }}{% endif %}" alt="User profile pictures">
+									{% endfor %}
+								{% else %}
+									Aucun salarié n'est impacté par cette décision pour l'instant.
+								{% endif %}
+							</div>
 						</div>
-					</div>
-					<div>
-						<div class="d-flex justify-content-center my-3">
-							<div class="d-none d-lg-block">
-								<h3 class="text-primary fw-bold text-center">Personnes expertes</h3>
-								<div class="d-flex justify-content-center my-3">
-									{% if expertUsers %}
-										{% for expertUser in expertUsers %}
-											<img class="arrow profil-icons" src="{% if expertUser.user.poster %}{{ asset('uploads/avatar/')~ expertUser.user.poster }}{% else %}{{ asset('build/images/icons/person.svg') }}{% endif %}" alt="User profile pictures">
-										{% endfor %}
-									{% else %}
-										Aucun salarié expert n'est attribué à cette décision pour l'instant.
-									{% endif %}
+						<div>
+							<div class="d-flex justify-content-center my-3">
+								<div class="d-none d-lg-block">
+									<h3 class="text-primary fw-bold text-center">Personnes expertes</h3>
+									<div class="d-flex justify-content-center my-3">
+										{% if expertUsers %}
+											{% for expertUser in expertUsers %}
+												<img class="arrow profil-icons" src="{% if expertUser.user.poster %}{{ asset('uploads/avatar/')~ expertUser.user.poster }}{% else %}{{ asset('build/images/icons/person.svg') }}{% endif %}" alt="User profile pictures">
```

Comment:
faire plutot un for / else (qui evite le if au dessus)
voir doc twig si besoin

Reply:
Cela arrive dans une autre PR